src/dotnet-svcutil/lib/tests/Baselines/Collections/dictionary/dictionary.csproj

```diff
@@ -3,15 +3,17 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>N.N</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="dotnet-svcutil-lib" Version="99.99.99" />
+    <Content CopyToOutputDirectory="always" Include="$(NuGetPackageRoot)dotnet-svcutil-lib/99.99.99/content/internalAssets/**" Link="internalAssets/%(RecursiveDir)%(Filename)%(Extension)" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.ServiceModel.*", Version="N.N.N">
     <PackageReference Include="System.ServiceModel.*", Version="N.N.N">
     <PackageReference Include="System.ServiceModel.*", Version="N.N.N">
     <PackageReference Include="System.ServiceModel.*", Version="N.N.N">
-    <PackageReference Include="System.ServiceModel.*", Version="N.N.N">
```

Comment:
I guess this is cut is because System.security and Duplex are gone.  What about the new packages, like NetNamedPipe?

Reply:
Yes, this change is because System.SerivceModel.Security and Duplex are gone and being replaced with System.ServiceModel.Primitives.  
System.ServiceModel.NetNamedPipe package is included since last release ([reference](https://github.com/dotnet/wcf/blob/98e3bd07ddc47b4f7e77874276e140406ff04f9e/src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs#L74)), so it doesn't reflect change for NetNamedPipe  here.